src/QueryFetcher.ts

```diff
@@ -106,8 +107,18 @@ export class QueryFetcher<TOperationType extends OperationType = OperationType>
         this.forceUpdate();
     };
 
-    fetch(query: OperationDescriptor, fetchPolicy: FetchPolicy, options: Options): void {
+    fetch(
+        query: OperationDescriptor,
+        fetchPolicy: FetchPolicy,
+        options: Options,
+        skip?: boolean,
```

Comment:
`skip` already comes in options, right? I personally not sure if it is worth adding new `skip` param.

Reply:
yes, you are right 👍 
I added a parameter so as to distinguish the two different options and make it easier (at least for me) to read the code:
https://github.com/relay-tools/relay-hooks/blob/ea5eb35c4a4909536f01f9a12089f9d31580a9bf/src/QueryFetcher.ts#L105

In this way it is more evident that the skip is managed only by the resolve function and not by the retry function